src/DashboardService.Business/ChangeLog/GetChangeLogCommand.cs

```diff
@@ -28,9 +28,9 @@ public class GetChangeLogCommand : IGetChangeLogCommand
     _responseCreator = responseCreator;
   }
 
-  public async Task<OperationResultResponse<ChangeLogInfo>> ExecuteAsync(Guid id, CancellationToken ct)
+  public async Task<OperationResultResponse<ChangeLogInfo>> ExecuteAsync(Guid changeLogId, CancellationToken ct)
```

Comment:
I don't see any use cases for it. The only case I see is getting all change logs by parent entity id

Reply:
++